cmd/helm/lint.go

```diff
@@ -51,6 +53,21 @@ func newLintCmd(out io.Writer) *cobra.Command {
 			if len(args) > 0 {
 				paths = args
 			}
+			if client.WithSubcharts {
+				for _, p := range paths {
+					filepath.Walk(filepath.Join(p, "/charts"), func(path string, info os.FileInfo, err error) error {
```

Comment:
the slash in `/charts` can be removed. `filepath.Join` will add those as needed.